UnitedIncome/components
# Docs

### Description
> Please provide a brief description of your changes below.

Adds an `Introduction` story that is focused on the purpose of the storybook.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

We should probably re-do the intro page for Storybook. I'll see how that's made tomorrow and get it updated.

### Comments
> Any additional comments that will aid with the review process.

I still need to make a few minor changes to the opening paragraph.
